### ISSUE TITLE
Baseline ++ validation wrongly reported in paper?

### ISSUE THREAD

**Issue by ghost** [wyharveychen/CloserLookFewShot#77]
In the paper, it seems you perform validation on novel classes for Baseline and Baseline++ too, as per the sentence
" To reduce the risk of over-fitting, we use the validation set to select the epoch or episode
with the best accuracy for all methods, including baseline and baseline++"

But looking at the code, it seems you actually do not validation and set acc=-1 for Baseline (see https://github.com/wyharveychen/CloserLookFewShot/blob/e03aca8a2d01c9b5861a5a816cd5d3fdfc47cd45/train.py#L39 and https://github.com/wyharveychen/CloserLookFewShot/blob/e03aca8a2d01c9b5861a5a816cd5d3fdfc47cd45/methods/baselinetrain.py#L80)

Could you explain?
